doc/GUIDE.md

```diff
@@ -2169,24 +2169,46 @@ users. Here's a quick rundown:
 
 ## Debugging
 
-The following command installs with profiling enabled:
+To profile a component of the current project, simply pass the `--profile`
+flag to `stack`. The `--profile` flag turns on the `--enable-library-profiling`
+and `--enable-executable-profiling` Cabal options _and_ passes the `+RTS -p`
```

Comment:
Actually it uses the new `--enable-profiling` flag for recent versions of Cabal, but I'm not sure it's worth mentioning here.